src/web_voice_processor.ts

```diff
@@ -303,7 +303,7 @@ export class WebVoiceProcessor {
     options: WebVoiceProcessorOptions,
   ) {
     if (navigator.mediaDevices === undefined) {
-      throw new Error("Audio recording is not allowed or disabled.");
+      throw new WvpError("NotAvailableError", "Audio recording is not allowed or disabled.");
```

Comment:
rather than create a new error for this one, maybe we use `DeviceMissingError`?

Reply:
this is different compared to `DeviceMissingError`. This is an error that happens like if you specifically disable mediaDevices, or like open some website in HTTP that is not local host. I think they should be separate errors.